librb/include/rb_dictionary.h

```diff
@@ -157,24 +157,12 @@ extern unsigned int rb_dictionary_size(rb_dictionary *dtree);
 void rb_dictionary_stats(rb_dictionary *dict, void (*cb)(const char *line, void *privdata), void *privdata);
 void rb_dictionary_stats_walk(void (*cb)(const char *line, void *privdata), void *privdata);
 
-#ifndef _WIN32
-
 #define RB_POINTER_TO_INT(x)		((int32_t) (long) (x))
 #define RB_INT_TO_POINTER(x)		((void *) (long) (int32_t) (x))
 
 #define RB_POINTER_TO_UINT(x)		((uint32_t) (unsigned long) (x))
 #define RB_UINT_TO_POINTER(x)		((void *) (unsigned long) (uint32_t) (x))
```

Comment:
These could be removed now and basically inlined with the code, but that might make the code less readable in the long run.